Reezan

Shinigami-7 opened the pull request:
commented each code

Shinigami-7 committed:
Update sel.py

Shinigami-7 committed:
Merge branch 'main' of https://github.com/Jyotsan-Hamal/Nabil-Stock-Time-series-Forcasting into reezan

Shinigami-7 committed:
adding comment

jyotsanh commented on Webscraping/sel.py:
```diff
@@ -1,25 +1,26 @@
 
 
-driver = webdriver.Firefox(options=options)
+driver = webdriver.Firefox(options=options) # ->it provide  argument to driver whic run browser in backgroung
```
# Mr.Reezan
### the  spelling of 'which' is mistake can you look into it.
### at line 13 comment section pls i ask you to check again

jyotsanh commented on Webscraping/sel.py:
```diff
@@ -55,18 +56,26 @@ def page_source_to_dataframe(page_html):
 button = driver.find_element(By.XPATH,price_history_xpath)
+# -> driver click button
 button.click()
+
```
## from line 53 i don't see the comments being used why is  that .

Shinigami-7 reviewed (commented):
thank you for pointing that out

Shinigami-7 committed:
updating comment

'which' spelling on line 13 has been corrected and comment has been updated below line 53

jyotsanh reviewed (approved):
it looks better now.
